editor/export/editor_export_platform.cpp

```diff
@@ -623,6 +922,65 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 	}
 
 	HashSet<String> features = get_features(p_preset, p_debug);
+	PackedStringArray features_psa;
+	for (const String &feature : features) {
+		features_psa.push_back(feature);
+	}
+
+	// Check if custom processing is needed
+	uint32_t custom_resources_hash = HASH_MURMUR3_SEED;
+	uint32_t custom_scene_hash = HASH_MURMUR3_SEED;
+
+	LocalVector<Ref<EditorExportPlugin>> customize_resources_plugins;
+	LocalVector<Ref<EditorExportPlugin>> customize_scenes_plugins;
+
+	for (int i = 0; i < export_plugins.size(); i++) {
+		if (export_plugins[i]->_begin_customize_resources(Ref<EditorExportPlatform>(this), features_psa)) {
+			customize_resources_plugins.push_back(export_plugins[i]);
+
+			custom_resources_hash = hash_murmur3_one_64(export_plugins[i]->_get_name().hash64(), custom_resources_hash);
+			uint64_t hash = export_plugins[i]->_get_customization_configuration_hash();
+			custom_resources_hash = hash_murmur3_one_64(hash, custom_resources_hash);
+		}
+		if (export_plugins[i]->_begin_customize_scenes(Ref<EditorExportPlatform>(this), features_psa)) {
+			customize_scenes_plugins.push_back(export_plugins[i]);
+
+			custom_resources_hash = hash_murmur3_one_64(export_plugins[i]->_get_name().hash64(), custom_resources_hash);
+			uint64_t hash = export_plugins[i]->_get_customization_configuration_hash();
+			custom_scene_hash = hash_murmur3_one_64(hash, custom_scene_hash);
+		}
+	}
+
+	HashMap<String, FileExportCache> export_cache;
+	String export_base_path = "res://.godot/exported/" + itos(custom_resources_hash);
+
+	bool convert_text_to_binary = GLOBAL_GET("editor/export/convert_text_resources_to_binary");
+
+	if (convert_text_to_binary || customize_resources_plugins.size() || customize_scenes_plugins.size()) {
+		// See if we have something to open
+		Ref<FileAccess> f = FileAccess::open(export_base_path.path_join("file_cache"), FileAccess::READ);
+		if (f.is_valid()) {
+			String l = f->get_line();
+			while (l != String()) {
+				Vector<String> fields = l.split("::");
+				if (fields.size() == 4) {
+					FileExportCache fec;
+					String path = fields[0];
+					fec.source_md5 = fields[1].strip_edges();
+					fec.source_modified_time = fields[2].strip_edges().to_int();
+					fec.saved_path = fields[3];
+					fec.used = false; // Assume unused until used.
+					export_cache[path] = fec;
+				}
+				l = f->get_line();
+			}
+		} else {
+			// create the path
+			Ref<DirAccess> d = DirAccess::create(DirAccess::ACCESS_RESOURCES);
+			d->change_dir("res://.godot");
```

Comment:
Remember that you should not hardcode `.godot` due to the `application/config/use_hidden_project_data_directory` option, which exposes it as `res://godot/`. You should use `ProjectSettings::get_project_data_dir_name()` instead.

I notice that you also did the same in the EditorBuildProfiles so it needs to be fixed too.